### ISSUE TITLE
Training on our data

### ISSUE THREAD

**Issue by 123alaa** [xingyizhou/CenterNet#44]
Hi, 
I did not understand from the documentation, your code read the the data from txt files or from json? Can i put my data (images and txt files) in the directory of coco data, and run the training.
My data have for each object (class_id xc yc w h) normalized, as darknet work for coco, How to convert to your format?

Best.

**Comment by xingyizhou** [comment]
Hi,
  The code reads data from json. You will need to convert the txt files into json. You can refer to [this script](https://github.com/xingyizhou/CenterNet/blob/master/src/tools/convert_kitti_to_coco.py), which converts kitti annotations in txt to json.

**Comment by hewumars** [comment]
@xingyizhou 
Whether it can achieve the same accuracy  without a pre-trained model when training on our data from scratch ？

**Comment by hewumars** [comment]
@xingyizhou 
2019-05-05-15-59: epoch: 16 |loss 1.058632 | hm_loss 0.735492 | wh_loss 1.267443 | off_loss 0.196395 | time 14.983333 |
what are the loss value converge？

**Comment by xingyizhou** [comment]
Our dla model is trained with Imagenet pretraining by default. Training from random initialization will be about 3AP lower than with imagenet pretraining (trained for 70 epochs). A good loss (with slight difference from the paper) looks like: |loss 1.590770 | hm_loss 1.183207 | wh_loss 1.969439 | off_loss 0.210620 | time 23.883333 | loss 1.704510 | hm_loss 1.274375 | wh_loss 2.205941 | off_loss 0.209541 | time 2.133333. The loss values are different on different datasets.

**Comment by hewumars** [comment]
@xingyizhou Thanks，the model is working.
